### DISCUSSION TITLE
childs and children in Layout.php

### DISCUSSION THREAD

**Discussion by patrick-rudolph-umusic** [pimcore/pimcore#13106] [Q&A / Support] [answered]
Is there a good reason why the children property of a Layout is copied over to the deprecated childs property?
In very complex layouts this causes a dramatic increase of the amount of data for loading a class definition. For example we have a class definition with a complex layout. In Pimcore V6.9 the json containing the class definition was around 150KB. Now it's 20MB large!!

https://github.com/pimcore/pimcore/blob/9149abbb9e821a6f5a33cf975a641b1967cdfcd1/models/DataObject/ClassDefinition/Layout.php#L477

**Comment by jdreesen** [comment, marked as answer]
Yes, it's a BC layer that will be removed with Pimcore 11.

**Reply by dvesh3** [reply]
please see also https://github.com/pimcore/pimcore/pull/11762#issuecomment-1080311167
